src/components/smart/OeZoneerderLayerswitcher.vue

```diff
@@ -0,0 +1,186 @@
+<template>
```

Comment:
ik kan enkel in de default story delagen wijzigen, bij de andere klapt de dialog toe zonder iets te doen